src/shims/intrinsics.rs

```diff
@@ -367,21 +383,27 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.write_scalar(Scalar::from_f64(res), dest)?;
             }
 
-            "exact_div" =>
-                this.exact_div(this.read_immediate(args[0])?, this.read_immediate(args[1])?, dest)?,
+            "exact_div" => {
+                check_arg_count(args, 2)?;
+                this.exact_div(this.read_immediate(args[0])?, this.read_immediate(args[1])?, dest)?
+            }
 
-            "forget" => {}
+            "forget" => {
+                check_arg_count(args, 0)?;
```

Comment:
I think this takes 1 arg, but if so it may not be covered by any test.

Reply:
Yes it does take 1 arg, of arbitrary type, that it does nothing with.
Could you add a test (maybe in a separate PR)?

Reply:
I added a test in https://github.com/rust-lang/miri/pull/1337.